app/scripts/proactive/model/Job.js

```diff
@@ -332,17 +334,23 @@ define(
         }
 
         if (this.attributes.hasOwnProperty('Variables')) {
-          var variables = this.attributes.Variables;
+          var variables;
+          if (this.attributes.hasOwnProperty('BackupVariables')) {
+            this.attributes.Variables = JSON.parse(JSON.stringify(this.attributes.BackupVariables));
+          } else {
+            this.attributes.BackupVariables = JSON.parse(JSON.stringify(this.attributes.Variables));
```

Comment:
Can you add comments saying why you need this.attributes.BackupVariables and what you're doing with it?

Reply:
done